.kokoro/dashboard.sh

```diff
@@ -20,9 +20,38 @@ scriptDir=$(realpath $(dirname "${BASH_SOURCE[0]}"))
 ## cd to the parent directory, i.e. the root of the git repo
 cd ${scriptDir}/..
 
+output="$scriptDir/dashboard/target/tmp/output.txt"
 ## Move into the dashboard directory
 cd dashboard/
 
 echo -e "\n******************** BUILDING THE DASHBOARD ********************"
 
 mvn --fail-at-end clean install
```

Comment:
This is running the unit tests for the dashboard, so if we're going to re-use this script, let's make this line skip tests (`-DskipTests=true`) and have a separate job type that runs the tests (e.g. `mvn test`).